$NetBSD: patch-aa,v 1.6 2010/11/07 23:23:56 wiz Exp $

errno isn't always an int. Let header file define it.

--- src/core-utils/gnc-gkeyfile-utils.c.orig	2007-04-28 23:16:20.000000000 +0000
+++ src/core-utils/gnc-gkeyfile-utils.c
@@ -303,7 +303,6 @@ gnc_key_file_save_to_file (const gchar *
 {
   gchar *contents;
   gint fd;
-  extern int errno;
   gint length;
   ssize_t written;
   gboolean success = TRUE;
